Makefile

```diff
@@ -106,7 +106,7 @@ build:
 	cargo build --no-default-features --features "${ENABLE_FEATURES}" -p cmd
 
 run:
-	cargo run --no-default-features --features  "${ENABLE_FEATURES}" -p cmd --bin tikv-server
+	cargo run --no-default-features --features  "${ENABLE_FEATURES}" -p cmd --bin tikv-server -- --pd-endpoints 127.0.0.1:2379
```

Comment:
I worry this can not work too, because tikv needs a config file. 

maybe we can remove `run` section directly? @aknuds1

Reply:
@siddontang Are you sure tikv-server needs a config file? I'm totally new to TiKV, but `make run` works for me after making this fix.

How should I run tikv-server otherwise? The example in docs/how-to/deploy/using-binary.md also says to run tikv-server without a config file: 
```
/bin/tikv-server --pd-endpoints="127.0.0.1:2379" \
                    --addr="127.0.0.1:20160" \
                    --data-dir=tikv1 \
                    --log-file=tikv1.log
```

Reply:
oh, interesting, I remember we must provide a config before.
/cc @zhangjinpeng1987

Reply:
but I still don't think adding `-- --pd-endpoints 127.0.0.1:2379` in makefile is a good choice. 

IMO, I prefer removing `run` here.

Reply:
@siddontang OK, I can change the PR to remove the `run` target?

Reply:
yes, I think it is fine

IMO, I prefer `make build` and start `./target/debug/tikv-server` direclty :-)

Reply:
Well, I prefer to keep the `run` target. If `make build` starts a tikv-server, it means a successful make build requires a running `pd-server`.

Reply:
@overvenus I think @siddontang meant to run `./target/debug/tikv-server` by hand after building with `make build`. I have been using `cargo run` to do the same thing :)

Reply:
yes, here I just want to remove `run` and let us start tikv-server manually.

Reply:
I think we can make configurations more friendly for dev usage. For example, `pd-endpoints=127.0.0.1:2379` can be a default config.

Reply:
@breeswish 
yes, we can do it, but here, I don't think `make run` makes any sense here. if I want to use another config or another PD, how do I use through `make run`? 

If we just for dev, running tikv-server directly is fine.

Reply:
@siddontang  `cargo run` won't work now I guess.. So there does exist some necessarily to provide a convenient way (for developers) to just run it.

Reply:
@breeswish @siddontang What do you think, should we remove the `make run` target and instead document how to launch tikv-server via `cargo run` (as `make run` does in this PR)?

Reply:
> So there does exist some necessarily to provide a convenient way (for developers) to just run it.

Just build and start it manually with ./target/debug/tikv-server or ./target/release/tikv-server